airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/SnowflakeDestination.java

```diff
@@ -10,24 +10,19 @@
 import io.airbyte.protocol.models.v0.AirbyteMessage;
 import io.airbyte.protocol.models.v0.ConfiguredAirbyteCatalog;
 
-import java.util.Collections;
-import java.util.List;
-import java.util.Optional;
-import java.util.Set;
 import java.util.concurrent.Executors;
 import java.util.concurrent.ScheduledExecutorService;
 import java.util.function.Consumer;
 import lombok.extern.slf4j.Slf4j;
 
+// TODO: Remove the Switching Destination from this class as part of code cleanup.
```

Comment:
Removing the switching destination will further simplify everything.

However, this produces code changes that are additive instead of the mostly deletes we have here.

My thought is to do this as a follow up to keep this PR related to removing GCS/S3 & cleaning up the spec.json, and thus easier to review.